modules/enabling-baseline-capability-set.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// *post_installation_configuration/cluster-capabilities.adoc
+
+[id="setting_baseline_capability_set_{context}"]
+= Enabling the cluster capabilities by setting baseline capability set
+
+As a cluster administrator, you can enable the capabilities by setting `baselineCapabilitySet`.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (oc).
+* You have installed the link:https://stedolan.github.io/jq/[jq package].
+
+.Procedure
+
+* To set the `baselineCapabilitySet` run the following command:
++
+[source,terminal]
+----
+$ oc patch clusterversion version --type merge -p '{"spec":{"capabilities":{"baselineCapabilitySet":"vCurrent"}}}' <1>
+----
++
+<1> For `baselineCapabilitySet` you can specify `vCurrent`, `v4.11`, or `none`.
++
+The following table describes the `baselineCapabilitySet` values description.
++
+.Cluster capabilities `baselineCapabilitySet` values description
+[cols=".^4,.^6a",options="header"]
+|===
+|Value|Description
+
+|`vCurrent`
+|Specify when you want to automatically add new capabilities as they become recommended.
+
+|`v4.11`
+|Specify when you want the capabilities defined in {product-title} 4.11 and not automatically enable capabilities, which might be introduced in later versions.
+
+|`none`
+|Specify when the other sets are too large, and you do not need any capabilities or want to fine-tune via `additionalEnabledCapabilities`.
+
+|===
+
+[NOTE]
+====
+The `baselineCapabilitySet` overrides if `additionalEnabledCapabilities` is present.
```

Comment:
Hmm, the note seems incorrect. additionalEnabledCapabilities set defines the extra capabilities in addition to baselineCapabilitySet. I'd suggest dropping it.